requirements/base/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Not a big deal, but updating your local venv to use Python 3.11 should fix this

Reply:
Ohh nice catch, the Python upgrade PR was merged recently. I'll fix this and then merge this PR to test the runner and close https://app.clickup.com/t/868bv3r6q.